### PR TITLE
fix: added new list of bootstrap governance members

### PR DESCRIPTION
**Pull request by babadofar**

### CODE REVIEW

File: governance/README.md [hunk 1]
```diff
@@ -1,19 +1,33 @@
...
-The bootstrap committee will be replaced by an elected Jenkins X steering committee.
+The Initial bootstrap committee had to be replaced by a new bootstrap committee.
+This bootstrap committee should be replaced by an elected Jenkins X steering committee in the future.
 
-### Members
+### Current Members
+
+| Name | Profile | Affiliation |
+| ---- | ------- | ----------- |
+| Ankit Muhaptra | [@ankitm123](https://github.com/ankitm123) | Independent |
```
Comment (ankitm123) [inline governance/README.md:12]: Ankit D Mohapatra

File: governance/README.md [hunk 2]
```diff
@@ -1,19 +1,33 @@
...
 
 | Name | Profile | Affiliation |
 | ---- | ------- | ----------- |
 | James Strachan | [@jstrachan](https://github.com/jstrachan) | CloudBees |
-| James Rawlings | [@rawlingsj](https://github.com/rawlingsj) | CloudBees |
 | Craig Barber | [@craigdbarber](https://github.com/craigdbarber) | Google |
 | Pete Muir | [@pmuir](https://github.com/pmuir) | CloudBees |
 | Mauricio Salatino | [@salaboy](https://github.com/Salaboy) | Camunda/LearnK8s |
 
+
```
Comment (ankitm123) [inline governance/README.md:28]: NIT: remove extra lines

File: governance/README.md (hunk 1 above)
Comment (babadofar) [reply governance/README.md]: 👍

File: TOC/README.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+## Initial Bootstrap Committee
+
+The Initial bootstrap committee had to be replaced by a new bootstrap committee.
+This bootstrap committee should be replaced by an elected Jenkins X TOC in the future.
+
+### Current Members
+
+| Name | Profile | Affiliation |
+| ---- | ------- | ----------- |
+| Ankit Muhaptra | [@ankitm123](https://github.com/ankitm123) |  Berkshire Grey |
```
Comment (ankitm123) [inline TOC/README.md:12]: Ankit D Mohapatra 😬

File: TOC/README.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+   * How is work tracked (eg, steering project board)
+   * Member note taker, or a neutral facilitator role that exists outside of the committee
+* Election process, including:
+   * Who is eligible to run as a candidate
+   * How candidates are nominated
+   * Special elections in the case someone resigns or is impeached
+   * Voter registration and requirements
+   * Committee company representation quotas
+ 
+ ### Transition Process
```
Comment (ankitm123) [inline TOC/README.md:70]: Do we need this section?

Comment (babadofar) [reply TOC/README.md]: Nah. deleted it

**Review by ankitm123 — APPROVED** [review]

**Review by tomhobson — APPROVED** [review]

**Pull request merged**